scrapers/Hentaied.yml

```diff
@@ -7,27 +7,36 @@ sceneByURL:
 xPathScrapers:
   sceneScraper:
     common:
-      $desc: "*[local-name()='p' or local-name()='h3' or local-name()='ol' or local-name()='ul']"
+      $desc: "*[local-name()='p' or local-name()='h3' or local-name()='ol' or local-name()='ul' or (local-name()='div' and @class='excerpt')]"
     scene:
       Title:
-        selector: //div[@class="post-content"]/h1
+        selector: //div[@class="innerflex"]//h1
       Date:
-        selector: //div/h2
+        selector: //div[contains(@class, "datesingle")]
         postProcess:
-          - replace:
-              - regex: 'PUBLISHED:\s*'
-                with: ""
-          - parseDate: 2006-01-02
+          # - replace:
+              # - regex: 'PUBLISHED:\s*'
+              #   with: ""
+          - parseDate: January 2, 2006
       Performers:
-        Name: //div[@class="tagsmodels"]/a
+        Name: //div[@class="innerflex"]//div[contains(@class,"tagsmodels")]/a
       Tags:
-        Name: //ul[@class="post-categories"]//a
+        Name:
+          selector: //ul[@class="post-categories"]//a
+          postProcess:
+            - replace:
+                - regex: 'Exclude'
+                  with: ''
       Details:
         selector: //div[@class="cont" or @class="column"]/$desc
         concat: "\n\n"
+        postProcess:
+          - replace:
+            - regex: 'Description|\(Read Less\)|\(Read more\)'
```

Comment:
For the `Description` regex i would use something more restrictive in case it is actually present in the details
eg `^Description\n+`

Reply:
Updated in 07cb7d2